x-pack/plugin/esql/qa/testFixtures/src/main/resources/dissect.csv-spec

```diff
@@ -145,6 +145,19 @@ full_name:keyword | emp_no:keyword | b:keyword
 Bezalel Simmel    | Bezalel        | Simmel 
 ;
 
+overwriteNameAfterSort#[skip:-8.13.0]
```

Comment:
I think we should backport this bugfix to 8.13, so I'm setting the skip labels accordingly.